modules/monitoring-creating-cross-project-alerting-rules-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-cross-project-alerting-rules-for-user-defined-projects_{context}"]
+= Creating cross-project alerting rules for user-defined projects
+
+You can create alerting rules for user-defined projects that are not bound to their project of origin by configuring a project in the `user-workload-monitoring-config` config map. This allows you to create generic alerting rules that get applied to multiple user-defined projects instead of having individual `PrometheusRule` objects in each user project.
+
+.Prerequisites
+
+ifndef::openshift-dedicated,openshift-rosa[]
+* If you are a cluster administrator, you have access to the cluster as a user with the `cluster-admin` cluster role.
+* If you are a non-administrator user, you have access to the cluster as a user with the following user roles:
+** The `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project to edit the `user-workload-monitoring-config` config map.
+** The `monitoring-rules-edit` cluster role for the project where you want to create an alerting rule.
+* A cluster administrator has enabled monitoring for user-defined projects.
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `dedicated-admin` role.
++
+[NOTE]
+====
+If you are a non-administrator user, you can still create cross-project alerting rules if you have the `monitoring-rules-edit` cluster role for the project where you want to create an alerting rule. However, that project needs to be configured in the `user-workload-monitoring-config` config map under the `namespacesWithoutLabelEnforcement` property, which can be done only by cluster administrators.
+====
+* The `user-workload-monitoring-config` `ConfigMap` object exists. This object is created by default when the cluster is created.
+endif::openshift-dedicated,openshift-rosa[]
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Edit the `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` project:
```

Comment:
Thank you for the information. Because the monitoring docs use the same strusture that I used here (you can see all the procedures in, for example, [this section](https://docs.openshift.com/container-platform/4.17/observability/monitoring/configuring-the-monitoring-stack.html#configuring-the-monitoring-stack_configuring-the-monitoring-stack)) I think I will keep it as it is now for consistency. Thank you!